.github/workflows/gh_pages.yml

```diff
@@ -0,0 +1,52 @@
+name: deploy-book
+
+# Run this when the master or main branch changes
+on:
+  push:
+    branches:
+    - main
```

Comment:
I know on other repo's we usually have a main and develop (or latest) version of the docs. Would it make sense to enable this for the develop branch as well?

Reply:
Nvm, clarified that this is a RTD capability and not available on GitHub pages.